sdk/pom.xml

```diff
@@ -455,7 +455,7 @@
     <dependency>
```

Comment:
Done.  There was javadoc in ProtoCoder.java, so main really did have a dep on this.  I've removed that so this can be a test-only dep.